src/infrastructure/services/api-clients.ts

```diff
@@ -56,7 +57,10 @@ export class GaiaServiceApiClient {
                 return Boolean(filename !== UPLOADABLE_JSON_FILES.CLIENT_CONFIG || data.indexOf("BlobNotFound") > 0 || data.indexOf("NoSuchKey") > 0);
             });
         } catch (error) {
-            throw ErrorHelper.getPackageError(null, PackageErrorCode.GaiaGetFileFailed, filename);
+            if (error instanceof PackageError) {
```

Comment:
@SanchayPahalwani should be catching the specific error `FileNotFound`

Reply:
Fixed